### DISCUSSION TITLE
Installation guide for windows?

### DISCUSSION THREAD

**Discussion by silversurface** [AdguardTeam/AdGuardHome#5928] [Q&A]
good morning all,

i used to be able to find the step by step installation guide for windows in the wiki, but i can't seem to find it anymore. can anyone help me find it? (native in windows, not via docker) thanks in advance!

**Comment by ainar-g** [comment]
https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started#official-releases

Is this not working for you?

**Reply by silversurface** [reply]
i am going to try this today, i am having to uninstall and reinstall adguard because of recent issues i've had with it, so i figured i'd wipe it clean anba start over. 

i'm hoping i won't have issues, but i was just looking for the previous step by step guide just in case

**Reply by chinaniupai** [reply]
What is written here is：    run:./AdGuardHome -s install
But it didn't write how to run。
./AdGuardHome - s install, how does this code run? Where do you want to input it?
